docs/source/llms/llm-evaluate/index.rst

```diff
@@ -494,6 +497,178 @@ up OpenAI authentication to run the code below.
             model_type="question-answering",
         )
 
+.. _llm-eval-model-endpoint:
+
+Evaluating with an MLflow Deployments Endpoint
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For MLflow >= 2.11.0, :py:func:`mlflow.evaluate()` supports evaluating a model endpoint by directly passing the MLflow Deployments endpoint URI to the ``model`` argument.
+This is particularly useful when you want to evaluate a deployed model hosted by a local `MLflow Deployments Server <../deployments/index.html>`_ or `Databricks Foundation Models API and External Models <https://docs.databricks.com/en/machine-learning/model-serving/score-foundation-models.html>`_, without implementing custom prediction logic to wrap it as an MLflow model or a python function.
+
+Please don't forget to set the target deployment client by using :py:func:`mlflow.deployments.set_deployments_target` before calling :py:func:`mlflow.evaluate()` with the endpoint URI, as shown in the example below. Otherwise, you will see an error message like ``MlflowException: No deployments target has been set...``.
+
+.. hint::
+
+    When you want to use an external endpoint **not** hosted by an MLflow Deployments Server or Databricks, you can create a custom Python function following the :ref:`Evaluating with a Custom Function <llm-eval-custom-function>` guide and use it as the ``model`` argument.
+
+Supported Input Data Formats
+****************************
+
+The input data can be either of the following format when using an URI of the MLflow Deployment Endpoint as the model:
+
+.. list-table::
+    :widths: 20 40 40
+    :header-rows: 1
+    :class: wrap-table
+
+    * - Data Format
+      - Example
+      - Additional Notes
+
+    * - A pandas DataFrame with a string column.
+      - 
+        .. code-block:: python
+
+            {
+                "inputs": [
+                    "What is MLflow?",
+                    "What is Spark?",
+                ]
+            }
+
+      - For this input format, MLflow will construct the appropriate request payload to the model endpoint type. For example, if your model is a chat endpoint (``llm/v1/chat``), MLflow will wrap your input string with the chat messages format like ``{"messages": [{"role": "user", "content": "What is MLflow?"}]}``. If you want to customize the request payload e.g. including system prompt, please use the next format.
+
+    * - A pandas DataFrame with a dictionary column.
```

Comment:
Can we bit a bit more specific in the description about what that dictionary column should contain? I don't think it should contain a `max_tokens` key, just the `messages` key for now.

Reply:
Hmm but if customer's data is fetched from logs/traces, they should contain the params as well? I think it doesn't harm allowing that but reduces preprocessing effort.

Reply:
Fair enough! Let's keep it :)